islandora_pdfjs.module

```diff
@@ -78,3 +78,78 @@ function islandora_pdfjs_viewer_callback(array $params, $fedora_object = NULL) {
     return theme('islandora_pdfjs', array('fedora_object' => $fedora_object, 'dsid' => $dsid));
   }
 }
+
+/**
+ * Implements hook_CMODEL_PID_islandora_solr_object_result_alter().
+ *
+ * Add page viewing fragment and search term to show all search results within
+ * book on page load.
+ */
+function islandora_pdfjs_islandora_bookCModel_islandora_solr_object_result_alter(&$search_results, $query_processor) {
```

Comment:
A bit confused here. Why the BookCmodel and/or not the PDF CMODEL?

Reply:
Hum. good point.. I guess we could do PDF too and a few others maybe.

Reply:
Was mainly copying IAV functionality to pdf.js. Will update pr.

Reply:
So.. Not sure how far to run with this. basically anything that matches https://github.com/Islandora/islandora_pdfjs/blob/7.x/islandora_pdfjs.module#L42-L47 could potentially use the pdf.js as a viewer. 

There's no islandora_get_viewer_id_by_cmodel or viewer id registry function so we'd need to store vars islandora_book_viewers, islandora_book_page_viewers, islandora_newspaper_issue_viewers, etc. to be able to only add the url params when the particular content model was using pdf.js

thoughts?